esmvalcore/cmor/check.py

```diff
@@ -123,7 +127,8 @@ def report_errors(self):
             msg = 'There were errors in variable {}:\n{}\nin cube:\n{}'
             msg = msg.format(self._cube.var_name, '\n '.join(self._errors),
                              self._cube)
-            raise CMORCheckError(msg)
+            if self._raise_exception:
```

Comment:
I would say that putting this condition here will affect the reporting of "regular" errors.  And for errors that have a flexible reporting this condition has already been checked at the report_strict_error function.  So I removed it to be safe. I am writing some tests now.

Reply:
OK @sloosvel if you are testing thoroughly then all's great since behaviors will emerge from testing. Cheers :beer: